### DISCUSSION TITLE
Welcome to Azure Netapp Files Discussions!

### DISCUSSION THREAD

**Discussion by seanluce** [microsoft/azure-netapp-files#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ProKashif** [comment]
I'm Kashif Ahmed and working as Mobile apps and React developer. So I don't have any experience in community discussion but want to have this. I will help here with my best knowledge.

**Comment by emilqnn** [comment]
Hi
